contracts/deployments/mainnet/.migrations.json

```diff
@@ -36,5 +36,6 @@
   "038_staking_patch": 1647020527,
   "039_wrapped_ousd": 1649704912,
   "041_stop_buyback_swaps": 1655503634,
-  "040_vault_peg_check": 1659100448
+  "040_vault_peg_check": 1659100448,
```

Comment:
super nit: since we are touching this file, let's re-order the entries so that 40 is before 41 :)

Reply:
Actually nevermind. It is ordered by block number which is what we want.